### PR TITLE
initial draft of home page: added pages and components folder

### LINKED ISSUE #11: Frontend: Add "Home" page

### PR DESCRIPTION
**Pull request by th3n1ghtmanc0m3th**
This is an initial draft of the Home Page

### CODE REVIEW

**Review by Viraj-Sonavane — DISMISSED** [review]

File: frontend/src/components/Post.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    transform: 'rotate(180deg)',
+  },
+  avatar: {
+    backgroundColor: red[500],
+  },
+}));
+
+//pass properties to ActivitiesPost function
+
+const ActivitiesPost = (props) => {
```
Comment (graydosh) [inline frontend/src/components/Post.js]: With our linting rules, all components need to use [prop destructuring](https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0). This makes it really easy to see what properties a component uses and makes code more readable.

Along with this, our linter also requires using [prop-types](https://www.npmjs.com/package/prop-types) to validate the types of the properties being passed to our components. This requires including `prop-types` and then defining the validation object before exporting the component. Refer to [the usage](https://www.npmjs.com/package/prop-types#usage) for help on this.

File: frontend/src/components/Post.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import CardContent from '@material-ui/core/CardContent';
+import CardActions from '@material-ui/core/CardActions';
+import Collapse from '@material-ui/core/Collapse';
+import Avatar from '@material-ui/core/Avatar';
+import IconButton from '@material-ui/core/IconButton';
+import Typography from '@material-ui/core/Typography';
+import { red } from '@material-ui/core/colors';
+import FavoriteIcon from '@material-ui/icons/Favorite';
+import ShareIcon from '@material-ui/icons/Share';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
```
Comment (graydosh) [inline frontend/src/components/Post.js]: Per linting rules, any unused variables should be removed before commiting.

File: frontend/src/pages/Home.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import Post from '../components/Post';
+import Container from '@material-ui/core/Container';
+
+const OurHome = () => (
+	<Container>
+	     <h1>This is the Home Page</h1>
+	     <Post title="Disc Golf at DeLaveaga" date="04/20/2021" image={{ url:"http://delaveagadiscgolf.com/wp-content/uploads/2011/07/15_Tee.jpg", title:"This is a photo of a spot on the course." }} desc="Come down to Delaveaga for a great game of Disc Golf."/>
+	     <Post title="Muay Thai Session" date="06/02/2021" image={{ url:"https://classpass-res.cloudinary.com/image/upload/f_auto,q_auto/a3pr36op4izqmtazy31g.png", title:"This is a photo of a muay thai session." }} desc="Come by for an hour Muay Thai session!"/>
+	     <Post title="2 Man Pickup" date="06/15/2021" image={{ url:"https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcSoDUsrjS1YMSMz8mkrrZkp1j7YjvMhUsPscg&usqp=CAU" }} desc="A classic 2-man pickup game."/>
+	     <Post title="Kettlebell Run" date="05/08/2021"/>
```
Comment (graydosh) [inline frontend/src/pages/Home.js]: With React components, we can actually break these across multiple lines for better readability.
So the first post can be changed to:
```js
    <Post
      title="Disc Golf at DeLaveaga"
      date="04/20/2021"
      image={{
        url: 'http://delaveagadiscgolf.com/wp-content/uploads/2011/07/15_Tee.jpg',
        title: 'This is a photo of a spot on the course.'
      }}
      desc="Come down to Delaveaga for a great game of Disc Golf."
    />
```

**Review by RetroJect — CHANGES_REQUESTED** [review]
There is a lot of linting changes that need to happen. I strongly recommend VSCode as it will automatically highlight your code that has linting problems. It can also suggest changes as well which is really helpful.
Unfortunately there isn't a way we can run linting on our files currently besides through an editor. VSCode requires installing the eslint extension as well to get linting to work properly. Let me know if you need help installing extension or anything else.

**Comment by graydosh** [comment]
@Viraj-Sonavane Please do not resolve conversations if the pull request is not assigned to you. It's how we can keep track of which request has been addressed or not.

**Comment by Viraj-Sonavane** [comment]
Yes. Sorry for that.

**Review by RetroJect — APPROVED** [review]

**Review by Zungrysoft — APPROVED** [review]
Looks good

**Pull request merged**